src/community/hz-cluster/src/main/java/org/geoserver/cluster/ConfigChangeEvent.java

```diff
@@ -95,6 +95,7 @@
     private static final long serialVersionUID = 1L;
 
     public enum Type {
+        PREADD,
```

Comment:
The GSIP needs to be update to include the definition of this event, and the community shoudl be inquired.

Reply:
understood, I will send out an email